packages/gatsby-cli/src/index.ts

```diff
@@ -17,17 +17,29 @@ if (useJsonLogger) {
 // Check if update is available
 updateNotifier({ pkg }).notify({ isGlobal: true })
 
-const MIN_NODE_VERSION = `>=8.0.0`
+const MIN_NODE_VERSION = `8.0.0`
 
-if (!semver.satisfies(process.version, MIN_NODE_VERSION)) {
+if (!semver.satisfies(process.version, `>=${MIN_NODE_VERSION}`)) {
   report.panic(
     report.stripIndent(`
-      Gatsby requires Node.js v8 or higher (you have ${process.version}).
+      Gatsby requires Node.js ${MIN_NODE_VERSION} or higher (you have ${process.version}).
       Upgrade Node to the latest stable release: https://gatsby.dev/upgrading-node-js
     `)
   )
 }
 
+const NEXT_MIN_NODE_VERSION = `10.13.0`
+
+if (!semver.satisfies(process.version, NEXT_MIN_NODE_VERSION)) {
```

Comment:
I guess you should also check that it satisfies min version, or you'll have two warnings.

Reply:
First one is a panic so it'll never reach this 😛

Reply:
ah ok.